addons/website_forum/tests/test_forum.py

```diff
@@ -198,10 +198,36 @@ def test_vote_crash(self):
 
     def test_vote(self):
         self.post.create_uid.karma = KARMA['ask']
-        self.user_portal.karma = KARMA['upv']
-        self.post.with_user(self.user_portal).vote(upvote=True)
+        self.user_portal.karma = KARMA['dwv']
+        initial_vote_count = self.post.vote_count
+        res = self.post.with_user(self.user_portal).vote(upvote=True)
```

Comment:
Can't we do a variable, like "post_as_portal = self.post.with_user(self.user_portal)" ? That way we don't have the with_user everywhere :)